documentation/asciidoc/computers/remote-access/find-your-ip-address.adoc

```diff
@@ -0,0 +1,225 @@
+IP4.ADDRESS[1]:                         127.0.0.1/8
+IP4.GATEWAY:                            --
+IP6.ADDRESS[1]:                         ::1/128
+IP6.GATEWAY:                            --
+
+GENERAL.DEVICE:                         p2p-dev-wlan0
+GENERAL.TYPE:                           wifi-p2p
+GENERAL.HWADDR:                         (unknown)
+GENERAL.MTU:                            0
+GENERAL.STATE:                          30 (disconnected)
+GENERAL.CONNECTION:                     --
+GENERAL.CON-PATH:                       --
+
+GENERAL.DEVICE:                         eth0
+GENERAL.TYPE:                           ethernet
+GENERAL.HWADDR:                         D0:3B:FF:41:AB:89
+GENERAL.MTU:                            1500
+GENERAL.STATE:                          20 (unavailable)
+GENERAL.CONNECTION:                     --
+GENERAL.CON-PATH:                       --
+WIRED-PROPERTIES.CARRIER:               off
+IP4.GATEWAY:                            --
+IP6.GATEWAY:                            --
+----
+
+This command outputs information about the various network interfaces accessible on your Raspberry Pi. Check the `GENERAL.TYPE` row to see which kind of network interface each block describes. For example, "ethernet" is the ethernet port on your device, and "wifi" refers to the WiFi chip built into some devices. You'll look at different blocks of output to find your IP address depending on the way your device accesses the internet:
+
+* if your device connects to the internet using WiFi, check the "wifi" block
+* if your device connects to the internet using the ethernet port, check the "eth0" block
+
+Once you've identified the correct network interface block, look for a field named `IP4.ADDRESS[1]` for an IPv4 address or `IP6.ADDRESS[1]` for an IPv6 address. You can ignore the trailing slash and number (e.g. `/24`) in those fields.
+
+In the example above, the Raspberry Pi uses WiFi to access the internet. Check the block where the `GENERAL.TYPE` field reads "wifi" to find the IP address. In this case, you can access this device using the IPv4 address in the `IP4.ADDRESS[1]` field: `192.168.1.42`.
+
+==== Resolving `raspberrypi.local` with mDNS
+
+On Raspberry Pi OS, multicast DNS is supported out-of-the-box by the Avahi service.
+
+If your device supports mDNS, you can reach your Raspberry Pi by using its hostname and the `.local` suffix.
+The default hostname on a fresh Raspberry Pi OS install is `raspberrypi`, so by default any Raspberry Pi running Raspberry Pi OS responds to:
+
+[,bash]
+----
+ping raspberrypi.local
+----
+
+If the Raspberry Pi is reachable, `ping` will show its IP address:
+
+----
+PING raspberrypi.local (192.168.1.131): 56 data bytes
+64 bytes from 192.168.1.131: icmp_seq=0 ttl=255 time=2.618 ms
+----
+
+If you change the system hostname of the Raspberry Pi (e.g., by editing `/etc/hostname`), Avahi will also change the `.local` mDNS address.
```

Comment:
Should also mention here about the hostname being configurable via Raspberry Pi Imager.

Reply:
The next paragraph mentions that you can change the hostname, that's good enough for me.

Reply:
With the difference being that Raspberry Pi Imager lets you configure the hostname _before_ you even flash the image to SD card, so no need to muck about with trying to change it after you've booted up the OS (which may be tricky, if you're running headless).